Makefile

```diff
@@ -93,3 +93,7 @@ semgrep:
 	$(eval DEV_REF := $(shell git rev-parse develop))
 	SEMGREP_REPO_NAME=ethereum-optimism/optimism semgrep ci --baseline-commit=$(DEV_REF)
 .PHONY: semgrep
+
+clean-node-modules:
+	rm -rf node_modules
```

Comment:
add `yarn clean`? `yarn clean` does more than just node modules

Reply:
Slight preference for not but it does make sense to do a full clean and rebuild after this

Reply:
Would approve a follow up PR